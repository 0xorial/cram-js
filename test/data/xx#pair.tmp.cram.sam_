@SQ	SN:xx	LN:20	M5:bbf4de6d8497a119dda6e074521643dc	UR:/Users/rbuels/dev/htslib/test/xx.fa
a1	99	xx	1	1	10M	=	11	20	AAAAAAAAAA	**********	MD:Z:10	NM:i:0
b1	99	xx	1	1	10M	=	11	20	AAAAAAAAAA	**********	MD:Z:10	NM:i:0
c1	99	xx	1	1	10M	=	11	20	AAAAAAAAAA	**********	MD:Z:10	NM:i:0
a1	147	xx	11	1	10M	=	1	-20	TTTTTTTTTT	**********	MD:Z:10	NM:i:0
b1	147	xx	11	1	10M	=	1	-20	TTTTTTTTTT	**********	MD:Z:10	NM:i:0
c1	147	xx	11	1	10M	=	1	-20	TTTTTTTTTT	**********	MD:Z:10	NM:i:0
