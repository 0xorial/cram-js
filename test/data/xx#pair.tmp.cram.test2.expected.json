[
  {
    "tags": {},
    "flags": 99,
    "cramFlags": 5,
    "sequenceId": 0,
    "readLength": 10,
    "alignmentStart": 1,
    "readGroupId": -1,
    "readName": "a1",
    "lengthOnRef": 10,
    "mappingQuality": 1,
    "qualityScores": [
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9
    ],
    "uniqueId": 397,
    "mate": {
      "sequenceId": 0,
      "alignmentStart": 11,
      "uniqueId": 400,
      "readName": "a1"
    },
    "templateLength": 20
  },
  {
    "tags": {},
    "flags": 147,
    "cramFlags": 1,
    "sequenceId": 0,
    "readLength": 10,
    "alignmentStart": 11,
    "readGroupId": -1,
    "readName": "a1",
    "lengthOnRef": 10,
    "mappingQuality": 1,
    "qualityScores": [
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9
    ],
    "uniqueId": 400,
    "mate": {
      "sequenceId": 0,
      "alignmentStart": 1,
      "uniqueId": 397,
      "readName": "a1"
    },
    "templateLength": 20
  },
  {
    "tags": {},
    "flags": 99,
    "cramFlags": 5,
    "sequenceId": 0,
    "readLength": 10,
    "alignmentStart": 1,
    "readGroupId": -1,
    "readName": "b1",
    "lengthOnRef": 10,
    "mappingQuality": 1,
    "qualityScores": [
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9
    ],
    "uniqueId": 398,
    "mate": {
      "sequenceId": 0,
      "alignmentStart": 11,
      "uniqueId": 401,
      "readName": "b1"
    },
    "templateLength": 20
  },
  {
    "tags": {},
    "flags": 147,
    "cramFlags": 1,
    "sequenceId": 0,
    "readLength": 10,
    "alignmentStart": 11,
    "readGroupId": -1,
    "readName": "b1",
    "lengthOnRef": 10,
    "mappingQuality": 1,
    "qualityScores": [
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9
    ],
    "uniqueId": 401,
    "mate": {
      "sequenceId": 0,
      "alignmentStart": 1,
      "uniqueId": 398,
      "readName": "b1"
    },
    "templateLength": 20
  },
  {
    "tags": {},
    "flags": 99,
    "cramFlags": 5,
    "sequenceId": 0,
    "readLength": 10,
    "alignmentStart": 1,
    "readGroupId": -1,
    "readName": "c1",
    "lengthOnRef": 10,
    "mappingQuality": 1,
    "qualityScores": [
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9
    ],
    "uniqueId": 399,
    "mate": {
      "sequenceId": 0,
      "alignmentStart": 11,
      "uniqueId": 402,
      "readName": "c1"
    },
    "templateLength": 20
  },
  {
    "tags": {},
    "flags": 147,
    "cramFlags": 1,
    "sequenceId": 0,
    "readLength": 10,
    "alignmentStart": 11,
    "readGroupId": -1,
    "readName": "c1",
    "lengthOnRef": 10,
    "mappingQuality": 1,
    "qualityScores": [
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9,
      9
    ],
    "uniqueId": 402,
    "mate": {
      "sequenceId": 0,
      "alignmentStart": 1,
      "uniqueId": 399,
      "readName": "c1"
    },
    "templateLength": 20
  }
]